types/src/main.ts

```diff
@@ -4,6 +4,10 @@
  * ------------------------------------------------------------------------------------------ */
 'use strict';
 
+import { PieceTreeTextBufferBuilder, PieceTreeBase, DefaultEndOfLine } from 'vscode-piece-tree';
+import { Range as PieceTreeRange } from 'vscode-piece-tree/out/common/range';
```

Comment:
I should probably rename the `out` folder to `lib` or `api`.

Reply:
Yea, if you wanna change this + bump the version, I can reinstall + fix.

Reply:
I pushed the fix directly to your PR ;)